tests/trio106.py

```diff
@@ -0,0 +1,13 @@
+import importlib
+
+import trio
+import trio as foo
+from trio import *  # noqa
+from trio import blah, open_file as foo  # noqa
```

Comment:
Ooh, this confused me for a minute.  Note to self: add a block comment below - "Note that our tests exercise the *Visitor* classes, without going through the noqa filter later in flake8 - so these suppressions are picked up by our project-wide linter stack but *not* the tests."

Reply:
Yeah it confused me too - had to double-check that the flake8 filters got applied when ran as a plugin.